### PR TITLE
GL-382: Exemption list UI

### PR DESCRIPTION
**Pull request by rasikasri**
### Jira link

HOTT-<TODO>

### What?

I have added/removed/altered:

- [ ] Added Admin UI screen to list the exemptions

### Why?

I am doing this because:

- Admin user should be able to manage exemption

### CODE REVIEW

File: app/controllers/green_lanes/exemptions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module GreenLanes
+  class ExemptionsController < AuthenticatedController
+    include XiOnly
+
+    before_action :disable_service_switching!
+    before_action :check_service
+    def index
```
Comment (jebw) [inline app/controllers/green_lanes/exemptions_controller.rb:7]: whitespace

File: app/views/green_lanes/exemptions/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  Manage Green Lanes Exemptions
+</h2>
+
+<%= link_to 'Add a Exemption', new_green_lanes_exemption_path, class: 'govuk-button' %>
+
+<% if @exemptions.any? %>
+  <table>
+    <thead>
+    <tr>
+      <th>ID</th>
```
Comment (jebw) [inline app/views/green_lanes/exemptions/index.html.erb:11]: Not sure we need to expose the exemptions id to the admin user - is it relevant? They will be thinking in terms of the code which is also unique?

**Review by jebw — DISMISSED** [review]
Looks good to me, theres a white space fix and I'm not sure we want the user caring about what the id of the exemption is - they'll be more focused on what the `code` is - maybe one to check with Gabriel though?

File: app/views/green_lanes/exemptions/index.html.erb (hunk 2 above)
Comment (rasikasri) [reply app/views/green_lanes/exemptions/index.html.erb]: Removed the id

**Review by alexdesi — APPROVED** [review]
:white_check_mark:

**Pull request merged**